src/components/ApiCard.tsx

```diff
@@ -9,29 +9,34 @@ interface Props {
 function ApiCard({ api }: Props) {
```

Comment:
In mobile view it feels like there is just a bit more padding in the left side than required.
<img width="413" alt="image" src="https://github.com/gdsc-fetju/api-collection/assets/76874341/f123ed26-e559-4c00-a593-34fc772755f5">